Cython/Compiler/Main.py

```diff
@@ -40,6 +40,8 @@
 
 verbose = 0
 
+standard_include_path = os.path.abspath(os.path.normpath(
```

Comment:
I don't think you need `os.path.normpath` here.

Reply:
ok